### ISSUE TITLE
Can't register two SqlServer liveness

### ISSUE THREAD

**Issue by eiximenis** [Xabaril/BeatPulse#21]
Trying to add two SqlServer liveness:

```
context.AddSqlServer("the-sql-server-connection-string");
context.AddSqlServer("the-sql-server-connection-string-2");
```

Generates an error complaining about the "sqlserver" path is already registered

**Comment by Xabaril** [comment]
Related with #22 
The issue is present and this solve it, but at this moment you need to implement this feature in all liveness. Probably is good to review the design to move this behavior into the LivenessContext instead on all liveness

**Comment by eiximenis** [comment]
Ok.
I can provide a PR to solve the problem gradually. Only that will take a bit longer :)

**Comment by unaizorrilla** [comment]
Fixed on 3537ca9c5a9a7563c73911c25b9021883e7b252d
